### PR TITLE
New Package: libu2f-server v1.0.1

### PR DESCRIPTION
**Pull request by the-maldridge**
Furthur package additions moving torwards U2F support in Void's PAM-stack.

### CODE REVIEW

**Comment by xtraeme** [comment]
`libu2f-server-devel` must be a symlink to `libu2f-server`.

**Comment by the-maldridge** [comment]
Lots of bugfixing later, I have the package in a buildable state.  Please check the contents of the update file, while it contains what I'm pretty sure its supposed to contain, I am not familiar with the update format.

File: srcpkgs/libu2f-server/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'libu2f-server'
+pkgname=libu2f-server
+version=1.0.1
+revision=1
+wrksrc="${pkgname}-${pkgname}-${version}"
+build_pie=yes
+build_style=gnu-configure
+configure_args="--enable-gtk-doc-html=no"
+hostmakedepends="automake libtool pkg-config gengetopt check help2man"
+makedepends="json-c-devel hidapi-devel libressl-devel check-devel"
```
Comment (xtraeme) [inline srcpkgs/libu2f-server/template]: check-devel is not really necessary, it's only used to run the test suite.

File: srcpkgs/libu2f-server/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'libu2f-server'
+pkgname=libu2f-server
+version=1.0.1
+revision=1
+wrksrc="${pkgname}-${pkgname}-${version}"
+build_pie=yes
+build_style=gnu-configure
+configure_args="--enable-gtk-doc-html=no"
+hostmakedepends="automake libtool pkg-config gengetopt check help2man"
```
Comment (xtraeme) [inline srcpkgs/libu2f-server/template]: You are disabling help2man but you've added help2man to hostmakedepends... remove it. ditto for check.

File: srcpkgs/libu2f-server/template [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+homepage="https://developers.yubico.com/libu2f-server/"
+distfiles="https://github.com/Yubico/libu2f-server/archive/libu2f-server-${version}.tar.gz"
+checksum="d87405ab7b271c3f674d64af8c878fdbf74d3b1d76b09de000c0d2cda5b44828"
+
+pre_configure() {
+	autoreconf -fi
+}
+
+post_install() {
+	       vlicense COPYING LICENSE
```
Comment (xtraeme) [inline srcpkgs/libu2f-server/template]: fix indentation.

**Pull request merged**